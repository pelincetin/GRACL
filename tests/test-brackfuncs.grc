int main() {
    IntTable it = createIntTable(10);
    Graph g = createGraph(10);
    Node n1 = g.createNode("Hello");
    it[n1] = 42;
    printi(it[n1]);
}