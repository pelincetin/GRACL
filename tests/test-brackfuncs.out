42
5.500000